packages/verifier/__mocks__/react-native-safe-area-context.ts

```diff
@@ -0,0 +1,8 @@
+export function useSafeArea() {
```

Comment:
I realize you just added these to make the test pass but just in case we haven't mentioned this yet:
The verifier is android-only because the SMS-related permissions it needs aren't available on iOS